Source/Core/DiscIO/WIABlob.cpp

```diff
@@ -1235,7 +1235,7 @@ static void RVZPack(const u8* in, OutputParametersEntry* out, u64 bytes_per_chun
       }
 
       const u64 junk_bytes = next_junk_end - current_offset;
-      if (junk_bytes > 0)
+      if (junk_bytes > 0 && seed)
```

Comment:
@JosJuice is this actually possible? tbh I didn't fully understand the function to reason if it's actually possible to reach here with `seed==nullptr`, and if it is, I'm not sure if it can be silently ignored or not?